presto-main/src/main/java/com/facebook/presto/operator/ExplainAnalyzeOperator.java

```diff
@@ -144,8 +155,21 @@ public Page getOutput()
         if (!hasFinalStageInfo(queryInfo.getOutputStage().get())) {
             return null;
         }
+        String plan;
+        switch (format) {
+            case TEXT:
+                plan = textDistributedPlan(queryInfo.getOutputStage().get().getSubStages().get(0), functionAndTypeManager, operatorContext.getSession(), verbose);
+                break;
+            case JSON:
+                plan = jsonDistributedPlan(queryInfo.getOutputStage().get().getSubStages().get(0), functionAndTypeManager, operatorContext.getSession());
+                break;
+            case GRAPHVIZ:
+                plan = graphvizDistributedPlan(queryInfo.getOutputStage().get().getSubStages().get(0), functionAndTypeManager, operatorContext.getSession());
```

Comment:
Since we are not changing the Graphviz output, should we skip supporting `EXPLAIN ANALYZE (FORMAT GRAPHIZ)` ? (the output is same as `EXPLAIN (FORMAT GRAPHIZ)`)  
IMO, it will only add to confusion for users

Reply:
Thanks for your opinion on this one. I had a similar thought but decided to just include it anyways. However, if you think it is confusing I will remove support for it

Reply:
Yes, lets skip it for now. We can enhance Graphviz output in a different PR (if there's interest) and add it back

Reply:
I don't think it's particularly difficult, but I'm not familiar with the graphviz format. It would just require some more time+effort to make it work properly